src/Components/Facility/AddInventoryForm.tsx

```diff
@@ -219,86 +222,75 @@ export const AddInventoryForm = (props: any) => {
 
   return (
     <div className="px-2">
```

Comment:
Remove this div before the Page component. `Page` component adds standard padding and makes it consistent with other pages.

Reply:
okay , i will remove this